moped-editor/src/views/projects/projectView/ProjectTeamTable.js

```diff
@@ -0,0 +1,226 @@
+import React from "react";
+import { useQuery, useMutation } from "@apollo/client";
+
+// Material
+import { CircularProgress, TextField } from "@material-ui/core";
+import MaterialTable from "material-table";
+import Autocomplete from "@material-ui/lab/Autocomplete";
+
+import {
+  TEAM_QUERY,
+  ADD_PROJECT_PERSONNEL,
+  UPDATE_PROJECT_PERSONNEL,
+} from "../../../queries/project";
+
+const ProjectTeamTable = ({
+  projectState,
+  setProjectState,
+  projectId = null,
+}) => {
+  const isNewProject = projectId === null;
+
+  const { loading, error, data, refetch } = useQuery(TEAM_QUERY, {
+    variables: { projectId },
+    fetchPolicy: "no-cache",
+  });
```

Comment:
I thought cache was no longer a problem? not a issue whatsoever, just trying to recall what the status is

Reply:
Yep, cache is not an issue for Apollo anymore, but Material Table breaks without this fetch policy. I'll take a look in the last PR of this series. Thanks for the heads up!